writing/index.md

```diff
@@ -259,6 +259,82 @@ julia> using Pluto
 julia> Pluto.run()
 ```
 
+## Markdown
+
+\tldr{Markdown is also a good fit for literate programming, and Quarto is an alternative to notebooks.}
+
+[Markdown](https://www.markdownguide.org/) is a markup language used to add formatting elements to plaintext text files.
+For example, to bold text one encapsulates words with double asterisks: `**bold text**` -> **bold text**.
+Markdown files can be opened by any text editor, such as VSCode, and are particularly useful when the file itself is not the final product.
+
+### Plain Text Markdown
+Plain text markdown files, which have the `.md` extension, are not used for interactive programming, meaning one cannot run code written in the file.
+Plain text markdown files are usually rendered into something else, such as but not limited to documents (e.g., PDF, Word), websites, presentations, and even books.
```

Comment:
I feel like this is a Quarto/Pandoc feature, not a Markdown feature. In my eyes, Markdown is just a markup language that is easy for humans to read and easy for machines to parse.

I suggest moving this to the Quarto section.

Reply:
Updated